recipes/meta.yaml

```diff
@@ -17,29 +19,30 @@ requirements:
     - pip
     - python
     - setuptools
+  # These differ slightly from requirements as some
+  # packages such as icoscp aren't available on conda
   run:
```

Comment:
Could we base the micromamba build more directly from the environment.yml file? Otherwise it feels like we're not properly testing our 'conda' setup within the CI.

Reply:
OK we're now parsing the environment file to read in the dependencies.

Reply:
I'll create a separate PR for building a conda environment from our environment file.